### ISSUE TITLE
How to get the QM9 test MAE

### ISSUE THREAD

**Issue by peachchange** [shehzaidi/pre-training-via-denoising#6]

**Comment by peachchange** [comment]
Your work has greatly inspired me. But in the process of fine-tuning, I can't find the place recorded by homo/lumo test mae. Excuse me, am I wrong in any step?

**Comment by peachchange** [comment]
Maybe I see. Does the test.y need to be multiplied by 1 million?

**Comment by shehzaidi** [comment]
Hi @taoquanjin, thanks for your interest! All the metrics are stored in a file called `metrics.csv` which should be in the `log_dir` specified for the experiment. The MAE loss for homo/lumo has eV as the units. Alternatively, if you're using weights & biases, all metrics should be logged onto the dashboard. Hope that helps!
